src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -214,14 +212,7 @@ private void validateName(final String name, final EventType eventType) throws I
     }
 
     private void validateSchemaChange(final EventType eventType, final EventType existingEventType) throws InvalidEventTypeException {
-        final boolean existingNonNullSchemaChanged =
-                existingEventType.getSchema() != null && !existingEventType.getSchema().equals(eventType.getSchema());
-
-        final boolean existingNullSchemaChanged =
-                existingEventType.getSchema() == null && eventType.getSchema() != null;
-
-
-        if (existingNonNullSchemaChanged || existingNullSchemaChanged) {
+        if (!existingEventType.getSchema().equals(eventType.getSchema())) {
```

Comment:
@rcillo I've checked, there are no event types on sandbox without a schema. Moreover, before your fix it was mandatory as well, so every event type should have a schema